protocol/src/main/java/org/opensearch/sql/protocol/response/format/Format.java

```diff
@@ -17,7 +17,12 @@ public enum Format {
   JDBC("jdbc"),
   CSV("csv"),
   RAW("raw"),
-  VIZ("viz");
+  VIZ("viz"),
+  // format of explain response
+  SIMPLE("simple"),
+  STANDARD("standard"),
+  CODEGEN("codegen");
```

Comment:
codegen is speific to calcite?  I propose use generic word,  `extended`

Reply:
done